catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -165,14 +172,20 @@ public void setResultShow(List<String> resultShow) {
 
     public void setAttributeAliases(List<String> attributeAliases) {
         this.attributeAliases = attributeAliases.stream()
-                .map(str -> str.split("="))
-                .collect(Collectors.toMap(list -> list[0].trim(), list -> list[1].trim()));
+                .map(str -> str.split("=", 2))
+                .collect(Collectors.toMap(list -> list[0].trim(), list -> list.length > 1 ? list[1].trim() : ""));
```

Comment:
:question: In the case that someone accidentally puts in an invalid value and it results in the list being only `1` in size do we still want to return an empty string (meaning do we want to silently ignore bad fields)? Or should we put in an explicit check to either filter out invalid values or throw an exception?

Not sure what our "preferred" course of action is (error reporting to admin UI not a strong point as of right now) but I have multiple times seen us prefer to throw an exception (and fail to save) rather than silently succeed and trim data. Interested if any others have thoughts on this.

Reply:
IMO, It may be best to filter out the attribute descriptions that are bad instead of throwing an exception for the whole config.

Reply:
So if we go with the option of just validating/filtering out bad data we should probably just add a log message that we filtered out an attribute (Maybe at debug? and probably a good idea to wrap it in a `LOGGER.isDebugEnabled()` since this is inside something being streamed)